src/workloads/scale/LargeScaleModel.yml

```diff
@@ -208,10 +157,12 @@ Actors:
   Phases:
   - {Nop: true}
   - {Nop: true}
-  - Duration: *Duration
-    TrackProportion: *TrackProportion
-    Document: *RollingDocument
-    GlobalRate: *RollingWrites
+  - ExternalPhaseConfig:
+      Path: ../../phases/scale/LargeScalePhases.yml
+      Key: RollingWriterCmd
+      Parameters:
+        TrackProportion: *TrackProportion
+        Writes: *RollingWrites
```

Comment:
This is fine. This phase configuration exposes a parameter called `Writes` which it uses for `GlobalRate`. I chose to make it like this because `GlobalRate` is used for reading and writing so these parameter names are more descriptive.